site/book/01-getting-started/02-quickstart.md

```diff
@@ -35,7 +35,7 @@ At this point, you typically want to customize the package. With kpt, you can us
 approaches depending on your use case.
 
 You may want to manually edit the files. For example, modify the value of `spec.replicas`
```

Comment:
I've already tagged nginx and wordpress to 0.1. No action required, just FYI.

Reply:
We should delete the tag and push it again. I have updated the deployment image to 1.14.1. I will have a follow up PR with image updated to 1.14.2 and then we can put v0.2 tag